src/View/Product/product.html.twig

```diff
@@ -60,64 +60,79 @@
 			</tr>
 		</tbody>
 	</table>
+	<h2>Envie de commander ?</h2>
 	<form>
-        <div>
-          <label for="nom">Nom:</label>
-          <input type="text" id="nom" name="nom">
-        </div>
-        <div>
-          <label for="prenom">Prénom:</label>
-          <input type="text" id="prenom" name="prenom">
-        </div>
-        <div>
-          <label for="email">Email:</label>
-          <input type="email" id="email" name="email">
-        </div>
-        <div>
-          <label for="telephone">Téléphone:</label>
-          <input type="number" id="telephone" name="telephone">
-        </div>
-        <div>
-          <label for="ville">Ville:</label>
-          <input type="text" id="ville" name="ville">
-        </div>
-        <div>
-          <label for="pays">Pays:</label>
-          <input type="text" id="pays" name="pays">
-        </div>
-        <div>
+		<div class="input-classique">
+			<label class="label-classique" for="nom">Nom:</label>
+		</div>
+		<div class="input-classique">
+			<input type="text" id="nom" name="nom">
+		</div>
+		<div class="input-classique">
+			<label class="label-classique" for="prenom">Prénom:</label>
+		</div>
+		<div class="input-classique">
+			<input type="text" id="prenom" name="prenom">
+		</div>
+		<div class="input-classique">
+			<label class="label-classique" for="email">Email:</label>
+		</div>
+		<div class="input-classique">
+			<input type="email" id="email" name="email">
+		</div>
+		<div class="input-classique">
+			<label class="label-classique" for="telephone">Téléphone:</label>
+		</div>
+		<div class="input-classique">
+			<input type="number" id="telephone" name="telephone">
+		</div>
+		<div class="input-classique">
+			<label class="label-classique" for="adresse">Adresse:</label>
+		</div>
+		<div class="input-classique">
+			<input type="text" id="adresse" name="adresse">
+		</div>
+		<div>
+			<fieldset>
+				<legend>Choix des produits:</legend>
+				<div>
+					<input type="checkbox" id="choix" name="choix" value="Fines de claires"/>
+					<label for="Fines de claires">Fines de claires</label>
```

Comment:
il y a un petit souci ici sur les `id` et les `for`. Ce n'est pas possible de mettre plusieurs fois le même `id` (tu as mis `choix` pour chacun des champs). Une solution :
`id="fine-de-claire"` avec `for="fine-de-claire"`, idem pour les autres 😉